mne/commands/mne_anonymize.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+"""Anonymize .fif file.
+You can do for example:
+$ mne anonymize -f sample_audvis_raw.fif
+"""
+
+# Authors : Dominik Krzeminski
+#           Luke Bloy <luke.bloy@gmail.com>
+
+import sys
+import mne
+import os
+
+ANONYMIZE_FILE_PREFIX = 'anon'
+
+
+def mne_anonymize(fif_fname, out_fname, dateback, keep_his, overwrite):
+    """Call *anonymize_info* on fif file and save.
+    Parameters
+    ----------
+    fif_fname : str
+        Raw fif File
+    out_fname : str
+        Output file name
+    overwrite : bool
+        Overwrite output file if it already exists
+    """
+    dir_name = os.path.split(fif_fname)[0]
+    fif_fname = os.path.basename(fif_fname)
+
+    try:
+        raw = mne.io.read_raw_fif(os.path.join(dir_name, fif_fname),
```

Comment:
it seems a little perverse to split the dirname from the basename only to re-join them a couple lines later. why not kill line 29 (where you overwrite the original `fif_fname` with its basename) and just use the original `fif_fname` here?  (and then use `os.path.basename` only where you need it, in line 42)